### PR TITLE
Setting active storage, display picture

### PR DESCRIPTION
**Pull request by isabloCh**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -29,6 +30,8 @@ gem 'jbuilder', '~> 2.5'
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
```
Comment (resivalex) [inline Gemfile]: Remove this line or move to the right place

File: app/admin/books.rb [hunk 2]
```diff
@@ -3,21 +3,47 @@
...
+      input :recommended_church
+      input :picture, as: :file
+    end
+    f.actions
+  end
+  
+  show do
+    attributes_table do
+      row :title
+      #ToDo в случае отсутствия картинки цеплять заглушку(нужно ли это в админке)?
```
Comment (resivalex) [inline app/admin/books.rb]: Strange todo style
`# TODO`

Comment (resivalex) [reply app/admin/books.rb]: Your question is about user experience. Do the way, you like

File: app/admin/books.rb [hunk 3]
```diff
@@ -3,21 +3,47 @@
...
+  end
 
   index do
     selectable_column
     id_column
     column :title
     column :price
     column :description
-    column :recommended_church
+    column :recommended_church
```
Comment (resivalex) [inline app/admin/books.rb]: Remove the space

File: config/database.yml [hunk 4]
```diff
@@ -7,7 +7,9 @@ default: &default
 
 development:
   <<: *default
-  database: ChristianBook_development
+  database: db_dev
+  username: dany
+  password: 1q2w3e
```
Comment (resivalex) [inline config/database.yml]: It's better not to store passwords on GitHub

E.g.
```
# config/database.example.yml
# copy to database.yml and set you values
...
```

File: config/storage.yml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+local:
+  service: Disk
+  root: <%= Rails.root.join("storage") %>
```
Comment (resivalex) [inline config/storage.yml]: No newline at the end of file

**Review by resivalex — CHANGES_REQUESTED** [review]
Remove binary files from the repository and add them to `.gitignore`

File: app/admin/books.rb [hunk 6]
```diff
@@ -3,21 +3,47 @@
   # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
   #
   # has_many :users, through: :ordered_books
-  permit_params :title, :description, :recommended_church, :price
+  permit_params :title, :description, :recommended_church, :price, :picture
```
Comment (Loremaster) [inline app/admin/books.rb:6]: @resivalex @isabloCh what bothers me is that I don't see any complaints from codeclimate for this code. I executed rubocop for this file:

```
$ rubocop app/admin/books.rb
Inspecting 1 file
C

Offenses:

app/admin/books.rb:1:1: C: Metrics/BlockLength: Block has too many lines. [36/25]
ActiveAdmin.register Book do ...
^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/admin/books.rb:9:14: C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
    f.inputs "Books" do
             ^^^^^^^
app/admin/books.rb:18:1: C: Layout/TrailingWhitespace: Trailing whitespace detected.
app/admin/books.rb:22:7: C: Layout/LeadingCommentSpace: Missing space after #.
      #ToDo в случае отсутствия картинки цеплять заглушку(нужно ли это в админке)?
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/admin/books.rb:22:8: C: Style/CommentAnnotation: Annotation keywords like ToDo should be all upper case, followed by a colon, and a space, then a note describing the problem.
      #ToDo в случае отсутствия картинки цеплять заглушку(нужно ли это в админке)?
       ^^^^^
app/admin/books.rb:22:13: C: Style/AsciiComments: Use only ascii symbols in comments.
      #ToDo в случае отсутствия картинки цеплять заглушку(нужно ли это в админке)?
            ^
app/admin/books.rb:38:31: C: Layout/TrailingWhitespace: Trailing whitespace detected.
    column :recommended_church 
                              ^
app/admin/books.rb:48:1: C: Layout/EmptyLinesAroundBlockBody: Extra empty line detected at block body end.

1 file inspected, 8 offenses detected
```

and you can see it shows issues. 

@resivalex do you think rubocop is not setup correctly for codeclimate? 

In our app setup looks like that:

```
version: 2
plugins:
  rubocop:
    enabled: true
    channel: rubocop-0-52
    config:
      file: .rubocop.yml
```

Comment (resivalex) [reply app/admin/books.rb:6]: I think the rubocop setup is ok. Config files `.rubocop.yml` and `.rubocop_todo.yml` allow breaking these rules

**Review by resivalex — APPROVED** [review]

File: app/admin/books.rb (hunk 6 above)
Comment (Loremaster) [reply app/admin/books.rb:6]: @resivalex ah I see!

**Comment by Loremaster** [comment]
@isabloCh don't forget to delete branch on github when you merge a PR :)

**Pull request merged**